src/Client/Pages/About.razor

```diff
@@ -0,0 +1,157 @@
+    falls under this category, as well as
+    <ItemLink Id="32837" PlaceholderText="Warglaive of Azzinoth" Colorize="true" Bracketize="true" LinkEnabled="true" />
+    (both halves) and
+    <ItemLink Id="34334" PlaceholderText="Thori'dal, the Stars' Fury" Colorize="true" Bracketize="true" LinkEnabled="true" />.
+</AboutQA>
+
+<AboutQA Question="Do I make a Loot List for 10-man raids?">
+    No. Loot Lists are only for 25 man raids. Karazhan and Zul'Aman are not part of any official raid
+    team's schedule.
+</AboutQA>
+
+<AboutQA Question="Can other players see my loot list?">
+    Your loot list ranks will be hidden until your list is approved and locked. Until then, other players can still view your list
+    but will not be able to see the exact rank you listed each item at. You can share your loot list with others if you wish,
+    but it is not recommended unless requesting help from others. The raid leaders will always be able to see your rankings.
+</AboutQA>
+
+<AboutQA Question="How do I win an item?">
+    The person with the highest priority when an item drops wins it. Priority is calculated by taking the rank
+    you gave an item and adding up all of the various bonuses you've acquired to it. The possible bonuses are:
+
+    <MudSimpleTable Elevation="4" Dense="true" Class="my-2">
+        <tbody>
+            <tr>
+                <td>Attendance</td>
+                <td>+1 for every 4 raids attended in the past 8 raid nights. Max +2.</td>
+            </tr>
+            <tr>
+                <td>Donations</td>
+                <td>+1 if the player donated at least 50 gold to the team the previous month.</td>
+            </tr>
+            <tr>
+                <td>Losses &amp; Passes</td>
+                <td>+1 each time the player does not win an item that drops while present.</td>
+            </tr>
+            <tr>
+                <td>Trial</td>
+                <td>-18 penalty for trial members the first week. Penalty reduced to -9 the second week.</td>
+            </tr>
+        </tbody>
+    </MudSimpleTable>
+</AboutQA>
+
+<AboutQA Question="What happens if someone has the same priority as me?">
+    If there's a tie for highest priority, then it goes to a <code>/roll</code> between those players. The highest
+    roll gets the item, and everyone else gets their loss bonus.
+</AboutQA>
+
+<AboutQA Question="Do I have to take an item if I have highest priority?">
+    No. Players may choose to pass to the next highest person or forego a tiebreaker roll, though there is no benefit
+    to doing this. You still gain your loss bonus as normal.
+</AboutQA>
+
+<AboutQA Question="What happens if an item drops and nobody has it on their list?">
+    Any item with no loot list competition, be it that everyone who listed it already has it or nobody in the entire
+    raid listed it, the item goes to an open <code>/roll</code> that anyone (even non-members) may roll for. The only
```

Comment:
"Any item with no loot list competition, be it that everyone who listed it already has it or nobody in the entire raid listed it, the item goes to an open"
remove "the item", reading the sentence without descriptor phrase ("be it that...raid listed it") doesn't make sense.

Reply:
"that anyone (even non-members) may"
change the parenthesis to commas

Reply:
The descriptor just seemed unnecessary after reading it. Removed it to be:
> Any item with no loot list competition goes to an open <code>/roll</code> that anyone, even non-members, may roll for. The only restriction to this roll is the item must be equipped by the winner.